src/classes/ElevateCaptureGroupService.cls

```diff
@@ -48,6 +48,37 @@ public virtual with sharing class ElevateCaptureGroupService {
         return new ElevateAuthorizedGift(tokenizedGift, authorizedGiftResponse);
     }
 
+    public ElevateCaptureGroup sendCreateRequest() {
```

Comment:
```suggestion
    public ElevateCaptureGroup getCaptureGroup() {
```

Reply:
I think we should stick closely to our already established language re capturing and capture groups etc